docker-compose.yaml

```diff
@@ -66,3 +67,24 @@ services:
     command: /bin/sh -c "python -u manage.py runserver 0.0.0.0:8000"
     environment:
       DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
+      ES_HOSTNAME: ${ES_HOSTNAME}
+
+  usaspending-es:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+    environment:
+    - cluster.name=usaspending
+    - network.host=0.0.0.0
+    - bootstrap.memory_lock=true
+    - "ES_JAVA_OPTS=-Xms1536m -Xmx1536m"
```

Comment:
Do we want to make this an env var? 
Should we highlight this in the documentation to ensure people are aware of the memory needs for this container?